### PR TITLE
Dummy app winter cleanup

### PR DESCRIPTION
**Pull request by nathalieArnoux**
### What does this PR do?

Related to: #[VEL-3968](https://linear.app/upfluence/issue/VEL-3968/rework-oss-components-dummy-app)

Complete reorganization of the OSS Components Dummy App into different pages for better organization and navigation.

Information on the repartition of the components is available in the spreadsheet [here](https://docs.google.com/spreadsheets/d/1Rqwrw3ECcYNQqnAy0LeqZVclDR_Y2aquTfLES4kT1w8/edit?usp=sharing).

Screenshot from 2024-12-13 11-18-50
Screenshot from 2024-12-13 11-19-35
Screenshot from 2024-12-13 11-19-41
Screenshot from 2024-12-13 11-19-54
Screenshot from 2024-12-13 11-20-04
Screenshot from 2024-12-13 11-20-13

### Good PR checklist
- [x] Title makes sense
- [x] Is against the correct branch
- [x] Only addresses one issue
- [x] Properly assigned
- [ ] Added/updated tests
- [ ] Added/updated documentation
- [ ] Migrated touched components to Glimmer Components
- [x] Properly labeled

### CODE REVIEW

File: tests/dummy/app/styles/app.less [hunk 1]
```diff
@@ -59,3 +59,16 @@
...
+
+.page-title-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: var(--color-white);
+  border: 1px solid var(--color-border-default);
+  border-radius: var(--border-radius-md);
+  padding: 12px;
+  margin: 18px auto 18px auto;
```
Comment (JulienVannier66) [inline tests/dummy/app/styles/app.less:71]:
```suggestion
  padding: var(--spacing-px-12);
  margin: var(--spacing-px-18) auto var(--spacing-px-18) auto;
```

File: tests/dummy/app/templates/input.hbs [hunk 2]
```diff
@@ -0,0 +1,454 @@
...
+        @onChange={{this.onCurrencyOnlyChange}}
+        @onlyCurrency={{true}}
+        @allowedCurrencies={{this.allowedCurrencies}}
+      />
+    </div>
+  </div>
+
+</div>
+
+{{!-- <div class="fx-col fx-gap-px-6 padding-px-18">
```
Comment (JulienVannier66) [inline tests/dummy/app/templates/input.hbs:394]: Do you want to keep all this part commented?

File: tests/dummy/app/templates/overlay.hbs [hunk 3]
```diff
@@ -0,0 +1,122 @@
+<div class="fx-col fx-gap-px-12 padding-px-18">
+  <div class="page-title-container">
+    <span class="font-size-h2 font-weight-semibold">Overlay</span>
+    <span class="font-color-gray-500">Components used to display information on the foreground</span>
+    <span class="font-color-gray-500 font-size-xs text-style-italic">(c'est qui qu'est over laid ?)</span>
```
Comment (JulienVannier66) [inline tests/dummy/app/templates/overlay.hbs:5]: :laughing:

**Review by JulienVannier66 — APPROVED** [review]
Otherwise LGTM

**Review by Miexil — APPROVED** [review]

**Pull request merged**